{{cookiecutter.project_slug}}/LICENSE

```diff
@@ -27,8 +27,6 @@ Copyright (c) {% now 'local', '%Y' %}, {{ cookiecutter.full_name }}
 
 Licensed under EUPL v1.2 or later ONLY
 
-Copyright (c) 2022, FERN.Lab
-
```

Comment:
I think it needs to stay, it is part of the License.  Replace 2022 by:
```
{% now 'utc', '%Y' %}
```

Cookiecutter will use the current year.